README.md

```diff
@@ -108,7 +110,7 @@ We are moving towards making more jobs to fit into the generated config.
 
 Presubmit will tell you if you forget to do any of this correctly.
 
-Merge your PR and @k8s-ci-robot will deploy your change automatically.
+Merge your PR and [@k8s-ci-robot](https://github.com/k8s-ci-robot) will deploy your change automatically.
```

Comment:
it still refuses to create a link?

Reply:
yeah, at least current https://github.com/kubernetes/test-infra does not have a link

Reply:
oh, I was looking at the wrong section in https://github.com/krzyzacy/test-infra/blob/ba4ac00d79243f52359ddcaa0d905f11cf89cc7e/README.md. 

there's another @k8s-ci-robot reference on L133 that may want to be updated, too.

Reply:
done